src/Services/Air/AirParser.js

```diff
@@ -403,9 +403,12 @@ function airPriceRspPricingSolutionXML(obj) {
     resultXml[root + '_XML'] = lines.join('\n');
   });
 
+  const segs = this.mergeLeafRecursive(objCopy, 'air:AirPriceRsp')['air:AirPriceRsp']['air:AirItinerary']['air:AirSegment'];
```

Comment:
please use the full name: `segments`

Reply:
segments was already used at the top of the function, so renamed it to mergedSegments

Reply:
sure, it is ok!